src/assets/config.cfg

```diff
@@ -0,0 +1,99 @@
+; Configuration file

```

Comment:
I tend to avoid putting configuration into source control, specifically as it makes it harder for other users to pick up your code and "just use it". Typically what I've seen is using a "example" or a template of the config and committing that to source control. Especially as this has username/password details.